### ISSUE TITLE
Unable to Import a Plugin

### ISSUE THREAD

**Issue by Ravin0319** [DivanteLtd/vue-storefront#2719]
Hi I am trying to import and use 'vue-instantsearch'
https://github.com/algolia/vue-instantsearch

I followed the steps here:-
https://docs.vuestorefront.io/guide/core-themes/plugins.html

`file: src/theme/test/index.js`
`import { setupMultistoreRoutes } from '@vue-storefront/core/lib/multistore`
`import App from './App.vue'`
`import routes from './router'`
`import Vue from 'vue'`
`import VueProgressBar from 'vue-progressbar'`
`import { isServer } from '@vue-storefront/core/helpers'`
`import InstantSearch from 'vue-instantsearch'`
`import '@vue-storefront/core/lib/passive-listeners'`
`Vue.use(VueProgressBar)`
`if (!isServer)Vue.use(InstantSearch)`

**I get the below error in Terminal:-**

`
Error during render : /
/Users/test/Sites/gc/apotheke/vue-storefront/node_modules/instantsearch.js/es/connectors/index.js:1
(function (exports, require, module, __filename, __dirname) { export { default as connectClearRefinements } from './clear-refinements/connectClearRefinements';
                                                              ^^^^^^

SyntaxError: Unexpected token export
    at new Script (vm.js:79:7)
    at createScript (vm.js:251:10)
    at Object.runInThisContext (vm.js:303:10)
    at Module._compile (internal/modules/cjs/loader.js:657:28)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.<anonymous> (/Users/test/Sites/gc/apotheke/vue-storefront/node_modules/vue-instantsearch/dist/vue-instantsearch.common.js:1:168)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
Error during render : /error
/Users/test/Sites/gc/apotheke/vue-storefront/node_modules/instantsearch.js/es/connectors/index.js:1
(function (exports, require, module, __filename, __dirname) { export { default as connectClearRefinements } from './clear-refinements/connectClearRefinements';
                                                              ^^^^^^
SyntaxError: Unexpected token export
    at new Script (vm.js:79:7)
    at createScript (vm.js:251:10)
    at Object.runInThisContext (vm.js:303:10)
    at Module._compile (internal/modules/cjs/loader.js:657:28)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.<anonymous> (/Users/test/Sites/gc/apotheke/vue-storefront/node_modules/vue-instantsearch/dist/vue-instantsearch.common.js:1:168)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)`

What am i doing wrong here? Please help me

**Comment by pkarw** [comment]
Please make sure that Your theme is properly processed by Babel to use ES5/6 -> https://github.com/DivanteLtd/vue-storefront/blob/159357fba37e1c9eecc1c4f128120d6964a64f24/core/build/webpack.base.config.ts#L143

Apparently InstantSearch module needs to be explicitely added to babel include

**Comment by Ravin0319** [comment]
Hey,

Thanks for responding.

I added this:-

include: [
          path.resolve(__dirname, '../../node_modules/@vue-storefront'),
          path.resolve(__dirname, '../../node_modules/vue-instantsearch'),
          path.resolve(__dirname, '../../src'),
          path.resolve(__dirname, '../../core')
        ]

Vue.use(InstantSearch)

and i still have the same Error.

If i add vue-instantsearch in include, Should i be using it in a different way?

Basically, I need the vue-instantsearch in global level, it will be under navigation like a search bar which is will be all pages.

**Comment by Ravin0319** [comment]
Any word on this?
